arangod/GeneralServer/VstCommTask.cpp

```diff
@@ -244,7 +244,7 @@ bool VstCommTask<T>::processMessage(velocypack::Buffer<uint8_t> buffer,
       RequestStatistics::SET_SUPERUSER(stat);
     }
 
-    LOG_TOPIC("92fd6", DEBUG, Logger::REQUESTS)
+    LOG_TOPIC("92fd6", INFO, Logger::REQUESTS)
```

Comment:
can't we keep this at DEBUG log level as before?

Reply:
no this is consistent with the per request logging in other protocol handlers

Reply:
vale